paddle/phi/backends/onednn/onednn_reuse.h

```diff
@@ -24,9 +24,12 @@ limitations under the License. */
 #include "paddle/phi/backends/onednn/onednn_context.h"
 #include "paddle/phi/backends/onednn/onednn_helper.h"
 #include "paddle/phi/common/data_type.h"
+#include "paddle/phi/common/int_array.h"
 #include "paddle/phi/common/place.h"
+#include "paddle/phi/common/scalar.h"
 #include "paddle/phi/core/dense_tensor.h"
 #include "paddle/phi/kernels/funcs/data_layout_transform.h"
+#include "paddle/phi/kernels/funcs/pooling.h"
 
 namespace phi {
 namespace funcs {
```

Comment:
I'm sorry that we forgot to modify the namespace when moved the `onednn_reuse.h` file from `phi/kernels/funcs` into `phi/bakcends/onednn`. we'd better unify it with the naming style of other backends like `phi/bakcends/xxx` later